src/components/table/CompanyRow.js

```diff
@@ -18,31 +19,45 @@ class CompanyRow extends React.Component {
     window.open(link);
   }
 
+  showLogo(link) {
```

Comment:
this could be a separate component for card view. so you could just import it into two files.